treedoc/tests/test_utils.py

```diff
@@ -164,6 +165,34 @@ def test_resolve_object():
     import collections.abc as module
 
     assert resolve_object("collections.abc") == module
+    
+    
+def test_signature():
+    ''' 
+    Test that formatting signature works.
+    '''
+    
+    def myfunc(a: float=4.2, b: int=42, *args, **kwargs):
```

Comment:
Comment:Having this order of `*args` and `**kwargs` makes little sense, although syntactically correct. Positional argument follows keyword argument, so you would do `func(arg1, arg2, arg3, *args, kwarg1=1, kwarg2=3, kwarg3=5, **kwrags)` instead.

The test case `def myfunc(a, b:int, *args, c=4.2, d:int=42, **kwargs)` captures the more common usage, as well as a mix of type annotations.